docs/redirects.yaml

```diff
@@ -0,0 +1,35 @@
+"en/latest/":
+    # Redirects in /latest/ introduced for 1.10.0 doc reorganization
+    "introduction/": {replace_key: ""}
+    "introduction/flocker-supports.html": {replace_key: "supported/"}
```

Comment:
So this means `docs.clusterhq.com/en/latest/introduction/flocker-supports.html` gets redirected to `docs.clusterhq.com/en/latest/supported/`? And I see `http://doc-dev.clusterhq.com/supported/` exists and that makes sense. OK.